### ISSUE TITLE
Empty fields lat, long and accurate in the EventLlog Table 

### ISSUE THREAD

**Issue by He3556** [SecUpwN/Android-IMSI-Catcher-Detector#637]
When a changing LAC detection takes place the lat, lng and accu fields are empty in the EvenLlog Table.

**Comment by SecUpwN** [comment]
@He3556, is this bug still present in our latest Buildozer build? Please help me to re-verify all open bugs!

**Comment by He3556** [comment]
yes but it is not a bug as far as i remember it is just not implemented.

**Comment by f3ndot** [comment]
The problem is `CellTracker#mMonitorCell` does not have its latitude, longitude set (or much, for that matter).

Using the `Cell` latitude/longitude setters somewhere in `CellTracker` with GPS coords *might* be an OK solution.

Using `AimsicdService#lastKnownLocation()` or `LocationTracker#lastKnownLocation()` is the golden ticket.
